### PR TITLE
DQA-4440: Drop grumphp dependency.

### PR DESCRIPTION
**Pull request by joaocsilva**
# NOTES

This PR needs a release on QA Automation and adapt `composer.json` to that version.
- https://github.com/ec-europa/qa-automation/pull/113 

Prepare release `8.7.0` https://github.com/ec-europa/toolkit/blob/77de747c013d31edf77c87d10887760f4dbfeebe/src/Toolkit.php#L15

### CODE REVIEW

**Comment by jonhy81** [comment]
Need to include the dependency to qa-automation 9.x

File: .drone.yml [hunk 1]
```diff
@@ -6,6 +6,18 @@ workspace:
   base: /test
   path: toolkit
 
+matrix:
+  PHP_VERSION:
+    - 7.4
```
Comment (jonhy81) [inline .drone.yml]: Remove 7.4 and 8.0 support, force in composer.

File: .drone.yml [hunk 2]
```diff
@@ -6,6 +6,18 @@ workspace:
...
+    - 8.0
+    - 8.1
+  COMPOSER_BOUNDARY:
+    - lowest
+    - highest
+  COMPOSER_VERSION:
+    - 1
```
Comment (jonhy81) [inline .drone.yml]: Composer 1 out

File: .drone.yml [hunk 3]
```diff
@@ -6,6 +6,18 @@ workspace:
...
+    - 2
+
 services:
   web:
     image: registry.fpfis.eu/fpfis/httpd-php:${PHP_VERSION=7.4}-ci
```
Comment (jonhy81) [inline .drone.yml]: Default to 8.1

File: composer.json [hunk 4]
```diff
@@ -19,7 +19,7 @@
         "behat/behat": "^3.7 || ^3.8",
         "cweagans/composer-patches": "^1.4 || ^1.7",
         "drush/drush": "^9.7.1 || ^10.0.0 || ^11.0.4",
-        "ec-europa/qa-automation": "^8.1.5",
+        "ec-europa/qa-automation": "dev-feature/DQA-4440",
```
Comment (jonhy81) [inline composer.json]: Set to newer 9.0

File: config/default.yml [hunk 5]
```diff
@@ -68,7 +68,7 @@ toolkit:
         - "/"
         - "/user"
     phpcs:
-      mode: grumphp
+      mode: phpcs
```
Comment (jonhy81) [inline config/default.yml:71]: We need to communicate this.

File: docker-compose.yml [hunk 6]
```diff
@@ -6,11 +6,12 @@ services:
     volumes:
     - ${PWD}:${PWD}
     - ${HOME}/.composer/auth.json:/root/.composer/auth.json
```
Comment (jonhy81) [inline docker-compose.yml]: Update to  ${HOME}/.config/composer/auth.json:/root/.config/composer/auth.json

File: docker-compose.yml [hunk 7]
```diff
@@ -6,11 +6,12 @@ services:
...
-    - ${PWD}/resources/xDebug/xdebug.custom.ini:/etc/php/8.1/mods-available/xdebug.ini
     environment:
       DOCUMENT_ROOT: ${PWD}
       ASDA_USER:
```
Comment (jonhy81) [inline docker-compose.yml]: Remove ASDA creds

File: docs/available-tasks.md [hunk 8]
```diff
@@ -5,57 +5,61 @@ To list all available tasks, please run:
 docker-composer exec web ./vendor/bin/run
 ```
 
-See bellow current list for version `8.6.19`.
+See bellow current list for version `8.7.0`.
```
Comment (jonhy81) [inline docs/available-tasks.md]: 9.0.0

File: docs/git-hooks.md [hunk 9]
```diff
@@ -0,0 +1,214 @@
...
+- [Configuring a project](/docs/configuring-project.md)
+- [Installing the project](/docs/installing-project.md)
+- [Testing the project](/docs/testing-project.md)
+- [Using Docker environment](/docs/docker-environment.md)
+- [Continuous integration](/docs/continuous-integration.md)
+- [Available tasks](/docs/available-tasks.md)
+- [Building assets](/docs/building-assets.md)
+- Git Hooks
+- [Update Project Documentation](/docs/project-documentation.md)
+- [Changelog](/CHANGELOG.md)
```
Comment (jonhy81) [inline docs/git-hooks.md:214]: Run phpdoc to update pages.

**Review by jonhy81 — CHANGES_REQUESTED** [review]

File: docs/git-hooks.md (hunk 9 above)
Comment (joaocsilva) [reply docs/git-hooks.md:214]: In the documentation was saying to run the phpdoc against the root of the project, this was making the docs to run in the `vendor` folder.
Changed it to run only on `src` folder

File: docker-compose.yml (hunk 6 above)
Comment (joaocsilva) [reply docker-compose.yml]: As there's no cross-platform solution I leave it commented out.

**Review by jonhy81 — APPROVED** [review]

**Pull request merged**